### PR TITLE
[MRG] Modif dataset neuropycon_data -> owncloud

### PR DESCRIPTION
**Pull request by davidmeunier79**

### CODE REVIEW

File: examples/plot_nii_to_graph.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
+"""
+.. _nii_to_graph:
+
+=========================================================
+Compute connecivity matrices and graph properties from nii files
+=========================================================
```
Comment (jasmainak) [inline examples/plot_nii_to_graph.py:6]: Nice, I didn't know you had started using sphinx gallery. That is great.

File: examples/plot_nii_to_graph.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+ROI_labels_file = op.join(data_path,"ROI_HCP","ROI_labels-ROI_HCP.txt")
+
+###############################################################################
+# Then, we create our workflow and specify the `base_dir` which tells
+# nipype the directory in which to store the outputs.
+
+# workflow directory within the `base_dir`
+conmat_analysis_name = 'conmat'
+
+from graphpype.pipelines.nii_to_conmat import create_pipeline_nii_to_conmat # noqa
```
Comment (jasmainak) [inline examples/plot_nii_to_graph.py]: you should change the namespace. Should be able to do:

```py
from graphpype.pipelines import create_pipeline_nii_to_conmat
```

might need to update the __init__.py to make it work

File: examples/plot_nii_to_graph.py [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+## for the analyses. con_den = 1.0 means a fully connected graphs (all edges
+## are present)
+
+# density of the threshold
+con_den = data_nii['con_den']
+
+# The optimisation sequence
+radatools_optim = data_nii['radatools_optim']
+
+from graphpype.pipelines.conmat_to_graph import create_pipeline_conmat_to_graph_density ## noqa
```
Comment (jasmainak) [inline examples/plot_nii_to_graph.py]: same comment here

File: examples/plot_nii_to_graph.py [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+## and visualize it. Take a moment to pause and notice how the connections
+## here correspond to how we connected the nodes.
+
+#from scipy.misc import imread  # noqa
+#import matplotlib.pyplot as plt  # noqa
+#img = plt.imread(op.join(data_path, graph_analysis_name, 'graph.png'))
+#plt.figure(figsize=(8, 8))
+#plt.imshow(img)
+#plt.axis('off')
+#plt.show()
```
Comment (jasmainak) [inline examples/plot_nii_to_graph.py]: why is this commented out?

File: examples/plot_nii_to_graph.py [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+The nii_to_graph pipeline performs graph analysis from functional MRI file
+in NIFTI format.
+
+The **input** data should be a preprocessed, and in the same space (e.g. MNI
+space) as the template used to define the nodes in the graph.
+"""
+
+# Authors: David Meunier <david_meunier_79@hotmail.fr>
+
+# License: BSD (3-clause)
```
Comment (jasmainak) [inline examples/plot_nii_to_graph.py:16]: you can add a comment like:

```py
# sphinx_gallery_thumbnail_number = 2
```

to choose which of your figures to show as thumbnail in the sphinx gallery

Comment (davidmeunier79) [reply examples/plot_nii_to_graph.py:16]: I did not know that, this is indeed much better!

File: examples/plot_nii_to_graph.py (hunk 4 above)
Comment (davidmeunier79) [reply examples/plot_nii_to_graph.py]: now running

File: examples/plot_nii_to_graph.py (hunk 3 above)
Comment (davidmeunier79) [reply examples/plot_nii_to_graph.py]: done

File: examples/plot_nii_to_graph.py (hunk 2 above)
Comment (davidmeunier79) [reply examples/plot_nii_to_graph.py]: done

File: examples/plot_nii_to_graph.py (hunk 1 above)
Comment (davidmeunier79) [reply examples/plot_nii_to_graph.py:6]: ;)

File: graphpype/utils_tests.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
+"""
+Support function for loading test datasets
+"""
+import os
+
+
+def load_test_data(name):
```
Comment (jasmainak) [inline graphpype/utils_tests.py:7]: There should be a docstring for any public function you add :)

**Review by jasmainak — APPROVED** [review]

File: .travis.yml [hunk 7]
```diff
@@ -43,7 +43,9 @@ install:
     # Testing dependencies
     #- pip install patsy
     - pip install codecov pytest pytest-cov;
-    - pip install neuropycon_data
+    #- pip install neuropycon_data
+    #- wget -O data_nii.zip --no-check-certificate  --content-disposition https://cloud.int.univ-amu.fr/index.php/s/STtqD3Y4ZZYqpTm/download
+    #- unzip -o data_nii.zip
```
Comment (jasmainak) [inline .travis.yml]: why is this commented out? Just remove it if you don't need it any more ...

File: graphpype/utils_tests.py (hunk 6 above)
Comment (davidmeunier79) [reply graphpype/utils_tests.py:7]: added

File: .travis.yml (hunk 7 above)
Comment (davidmeunier79) [reply .travis.yml]: done

**Comment by jasmainak** [comment]
@davidmeunier79 merge if you are happy!

**Pull request merged**